### PR TITLE
1125 add test for book.

### PR DESCRIPTION
**Pull request by Vivi202008**
CheckBook.js test a book from loggin to book and pay direktly.
CheckBook_Newbankcard, user has not bankcard in his count, so input information of bankcard.

### CODE REVIEW

File: CheckBook_Newbankcard.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    cy.get('.col-sm-2 > .btn').eq(0).click();
+    cy.get('.media-heading > a').should('contain','Lundby-Biskopsgården Klubb');
+      
+      cy.wait(100)
+      //choose time and slots          
+      cy.get('.no-margin.no-padding').first().click()  
+      cy.get('.slot.free').first().click()
+        //cy.get('.list-inline.no-margin > :nth-child(2) > .btn').contains('12').click({force: true}) 
+        //cy.get('#se4fabd4d75f9e0b00175fa5531aa04f7').click({force: true}) // book slot id
+        //cy.get('.col-sm-8 > :nth-child(3) > .radio > label').click({ force: true}) // payment method
```
Comment (ovelarsson) [inline CheckBook_Newbankcard.js]: Please remove this if it's not going to be used. it is so much cleaner and better to not include commented code (unless it's something temporal that will change later on.)

File: CheckBook_Newbankcard.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        cy.get('#btnSubmit').click() // submit payment
```
Comment (ovelarsson) [inline CheckBook_Newbankcard.js]: Please use consistency when adding comments.

File: CheckBook_Newbankcard.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+        cy.get(':nth-child(2) > .has-feedback').type("4111111111111111")
+        cy.get('.col-sm-10 > :nth-child(2) > :nth-child(2)').click().type("Jonas")
+      
+        cy.get(':nth-child(3) > .form-control').eq(0).select('03',{force:true})
+        cy.get('#expiryYear').eq(0).select('2030',{force:true})
+        cy.get('.col-sm-10 > :nth-child(2) > :nth-child(5)').type("737")
+
+        cy.get('#adyen-encrypted-form > .modal-footer > .btn-success').click()  //for first time
```
Comment (ovelarsson) [inline CheckBook_Newbankcard.js]: Make sure comments are displayed in the same way in all places. That way it's easier for the user to read.

**Review by ovelarsson — CHANGES_REQUESTED** [review]

File: CheckBook_Newbankcard.js (hunk 3 above)
Comment (mmfaiz) [reply CheckBook_Newbankcard.js]: ok

**Pull request merged**